From b81b35fc7696594eb257a0286180c882a77c7342 Mon Sep 17 00:00:00 2001
From: sai surya teja <sssteja@habana.ai>
Date: Tue, 07 Mar 2023 06:49:10 +0200
Subject: [PATCH] [SW-72653] gaudi3: ppboot: add bootrom export elf to ppboot repo

This patch adds bootrom exports elf file for asic, pldm and onix targets
in ppboot repo. Since, exports elf is common for asic and pldm targets
using a single elf for ppboot build.

This is needed as the gaudi3 bootrom development is frozen and a final
export elf file is copied to ppboot repo for its compilation.

Change-Id: I908d0d63316b26c73c381fae7cad60956dd5516e
---

diff --git a/arch/gaudi3/arc/exports.elf b/arch/gaudi3/arc/exports.elf
new file mode 100644
index 0000000..2e6517d
--- /dev/null
+++ b/arch/gaudi3/arc/exports.elf
Binary files differ
diff --git a/arch/gaudi3/arc/exports_onix.elf b/arch/gaudi3/arc/exports_onix.elf
new file mode 100644
index 0000000..de2eb48
--- /dev/null
+++ b/arch/gaudi3/arc/exports_onix.elf
Binary files differ
